requirements.txt

```diff
@@ -8,7 +8,7 @@ g2p>=0.5.20210825
 lxml==4.6.5
 networkx==2.5
 numpy>=1.16.4
-soundswallower==0.1.5
+soundswallower~=0.2
```

Comment:
What does this `~=` syntax mean?

Reply:
Hi! It means "compatible with" - so 0.2.1 will also be accepted, presuming that I practice proper semantic versioning and all 0.2.x are backwards-compatible with 0.2

Reply:
Ahh.  I admit not having read the specification very closely, thanks for your attention to this!  I would prefer to restrict to 0.2.* so looks like ~=0.2.0 is the right one to use.